schemas/rdf/shacl-schema.ttl

```diff
@@ -99,11 +99,11 @@ aas:AccessControlPolicyPointsShape a sh:NodeShape ;
     ] ;
   sh:property [
       a sh:PropertyShape ;
-      sh:path <https://admin-shell.io/aas/3/0/RC01/AccessControlPolicyPoints/policyEnforcementPoint> ;
-      sh:class aas:PolicyEnforcementPoint ;
+      sh:path <https://admin-shell.io/aas/3/0/RC01/AccessControlPolicyPoints/policyEnforcementPoints> ;
```

Comment:
policyEnforcementPoint in (https://admin-shell.io/aas/3/0/RC01/AccessControlPolicyPoints/policyEnforcementPoint) the property path should be in a single form.